README.md

```diff
@@ -103,14 +103,18 @@ in which a feature is introduced may change.
 SNAPSHOTS, PREVIOUS RELEASES AND SOURCE CODE
 --------------------------------------------
 Periodically development code snapshots are provided at the following URL:
-    
+
    https://github.com/HDFGroup/hdf5/releases/tag/snapshot
 
 Source packages for current and previous releases are located at:
-    
-   https://support.hdfgroup.org/downloads/hdf5
+
+   hdf5 1.14 releases:
+   https://support.hdfgroup.org/releases/hdf5/downloads
+
```

Comment:
Is this URL provisional?

I get:

`This page doesn’t seem to exist`

Reply:
I do too.  Maybe https://support.hdfgroup.org/downloads/index.html?